### PR TITLE
Improve Crashlytics Beta action documentation

### PR DESCRIPTION
**Pull request by mfurtak**
Provide more context and examples for the proper way to use the `groups` parameter, since there seems to be confusion about what to provide in that value

### CODE REVIEW

File: fastlane/docs/Actions.md [hunk 1]
```diff
@@ -876,8 +876,34 @@ crashlytics(
...
 )
 ```
+
 Additionally you can specify `notes`, `emails`, `groups` and `notifications`.
 
+#### Distributing to Groups
+
+When using the `groups` parameter, it's important to use the group **alias** names for each group you'd like to distribute to. A group's alias can be found in the web UI. If you're viewing the Beta page, you can open the groups dialog here:
+
+![Crashlytics Beta Groups Navigation](../assets/Beta-Groups-Nav.png)
```
Comment (asfalcone) [inline fastlane/docs/Actions.md]: Is it possible to put these images side by side using HTML? This adds quite a bit to the documentation here.

**Comment by asfalcone** [comment]
Great instructions :+1: Added one comment you might consider.

**Comment by mfurtak** [comment]
I wasn't able to find a great way to do this. If we discover a way to make it go horizontal and look good, I'll be happy to update it. :+1:

**Comment by KrauseFx** [comment]
@mfurtak You can just use `` tags and even `<p align="right">` and similar. Right now we're using a lot of space with those screenshots. Do you want to take a look at this, or should I prepare a PR?

**Comment by mfurtak** [comment]
Go for it! I'll learn from whatever you put up.
On Apr 18, 2016 5:38 AM, "Felix Krause" <notifications@github.com> wrote:

@mfurtak <https://github.com/mfurtak> You can just use 
tags and even <p align="right"> and similar. Right now we're using a lot of
space with those screenshots. Do you want to take a look at this, or should
I prepare a PR?

—
You are receiving this because you were mentioned.

Reply to this email directly or view it on GitHub
<https://github.com/fastlane/fastlane/pull/4213#issuecomment-211297071>

**Pull request merged**